android/lib/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -506,6 +508,39 @@ public void fitToElements(boolean animated) {
         }
     }
 
+    public void fitToSuppliedMarkers(ReadableArray markerIDsArray, boolean animated) {
+        LatLngBounds.Builder builder = new LatLngBounds.Builder();
+
+        String[] markerIDs = new String[markerIDsArray.size()];
+        for (int i = 0; i < markerIDsArray.size(); i++) {
+            markerIDs[i] = markerIDsArray.getString(i);
+        }
+
+        boolean addedPosition = false;
+
+        for (AirMapFeature feature : features) {
+            if (feature instanceof AirMapMarker) {
+                String identifier = ((AirMapMarker)feature).getIdentifier();
+                Marker marker = (Marker)feature.getFeature();
+                if (Arrays.asList(markerIDs).contains(identifier)) {
```

Comment:
extract `Arrays.asList(markerIDs)` as a variable outside of the loop so you dont allocate a new array every time